### PR TITLE
Support change log-level in runtime

### LINKED ISSUE #1610: Please support change log-level in runtime
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Motivation

version 2.5.0
error: Unsupported CONFIG parameter: log-level

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by enjoy-binbin**
Now we support config set log-level xxx to modify log-level
in runtime.

This closes #1610.

### CODE REVIEW

**Review by git-hulk — APPROVED** [review]

File: src/config/config.cc [hunk 1]
```diff
@@ -478,6 +478,12 @@ void Config::initFieldCallback() {
          if (cluster_enabled) srv->slot_migrator->SetSequenceGapLimit(sequence_gap);
          return Status::OK();
        }},
+      {"log-level",
+       [this](Server *srv, const std::string &k, const std::string &v) -> Status {
+         if (!srv) return Status::OK();
+         FLAGS_minloglevel = log_level;
```
Comment (mapleFU) [inline src/config/config.cc:484]: just a nit, should we not change it when `FLAGS_minloglevel` and `log_level` is equal?

Comment (enjoy-binbin) [reply src/config/config.cc:484]: i am not very familiar with this (the details), if it's the same, is there any side effect of changing it?

Comment (mapleFU) [reply src/config/config.cc:484]: Not so important, it will write a global variable, causing some cache invalid. But it's not important

Comment (enjoy-binbin) [reply src/config/config.cc:484]: ohh, i got it, this is new to me, thanks for the explanation.

**Pull request merged**